main.tf

```
@@ -41,25 +41,25 @@ resource "random_string" "deployment_id" {
 }
 
 resource "azurerm_resource_group" "aap" {
-  name     = var.resource_group
+  name = var.resource_group
   location = var.location
-  tags     = local.persistent_tags
+  tags = local.persistent_tags
 }
 
 resource "azurerm_virtual_network" "aap" {
-  name                = "${var.deployment_id}-aap-vn"
-  location            = azurerm_resource_group.aap.location
+  name = "${var.deployment_id}-aap-vn"
```

If we're doing `subnet` instead of `sn`, can this be `vnet`?
```suggestion
  name = "${var.deployment_id}-aap-vnet"
```